CMakeLists.txt

```diff
@@ -949,20 +961,18 @@ else ()
   endif()
 
   if (BUILD_STATIC)
-    add_library (libzmq-static STATIC $<TARGET_OBJECTS:objects> ${public_headers} ${html-docs} ${readme-docs} ${zmq-pkgconfig} ${CMAKE_CURRENT_BINARY_DIR}/version.rc)
+    if (MINGW)
+      add_library (libzmq-static STATIC ${sources} ${public_headers} ${html-docs} ${readme-docs} ${zmq-pkgconfig} ${CMAKE_CURRENT_BINARY_DIR}/version.rc)
+    else ()
+      add_library (libzmq-static STATIC $<TARGET_OBJECTS:objects> ${public_headers} ${html-docs} ${readme-docs} ${zmq-pkgconfig} ${CMAKE_CURRENT_BINARY_DIR}/version.rc)
+    endif ()
     set_target_properties (libzmq-static PROPERTIES
       PUBLIC_HEADER "${public_headers}"
       COMPILE_DEFINITIONS "ZMQ_STATIC"
       OUTPUT_NAME "zmq"
       PREFIX "lib")
   endif()
 
-  target_include_directories (objects
```

Comment:
why is this being removed?

Reply:
It's moved up to where the `add_library(objects ...` line is.

Reply:
Ah sorry, missed that